_pages/en/about-us.html

```diff
@@ -5,7 +5,12 @@
 lang: en
 permalink: /about-us.html
 ---
-
+<div class="team-intro">
+<p>This website is a platform where we, IT Strategy, are sharing our plan for becoming a digital government, our progress, and all the tools, information and other resources needed to support this transformation across IITB and ESDC as a whole. It is a place that gives us an opportunity to express our professional opinion and ideas.</p>
```

Comment:
I suggest to spell out ESDC and IITB acronyms, and provide context that IITB is our Branch.

Reply:
Agreed, done